FWCore/Integration/plugins/PostInsertProducer.cc

```diff
@@ -0,0 +1,85 @@
+#include <stdexcept>
+
```

Comment:
This `#include` doesn't seem to be needed
```suggestion
```

Reply:
Removed.